README.md

```diff
@@ -133,6 +133,7 @@
 ### Operating System
 
 - [clipboard](https://github.com/vlang/v/tree/master/vlib/clipboard) - V module for interacting with the OS clipboard. Fully cross-platform.
+- [vlipboard](https://github.com/asvvvad/vlipboard) - An easy to use wrapper of clipboard with Wayland and Termux support
```

Comment:
Please add a dot symbol at the end of the sentence to pass tests.

Reply:
Fixed. sorry about that it's 9am and still haven't slept good night!